### ISSUE TITLE
Docker image not working with alternate gui

### ISSUE THREAD

**Issue by H256** [H256/gekko#1]
**I'm submitting a ...**
[x] bug report
[ ] question about the decisions made in the repository

**Action taken** (what you did)
from https://github.com/H256/gekko-quasar-ui/issues/18#issuecomment-464509210
However I do have an ongoing issue, I was never able to run the gekko on server(even normal gui),
The only way I was able to accomplish it was through docker, I wish to run your GUI on server, but when I run docker image only normal GUI works, If I use your repo, It does not.

I have your GUI working on my desktop, and have one instance Gekko running on server, via Docker
Thank you

**Expected result** (what you hoped would happen)

**Actual result** (unexpected outcome)

**Other information** (e.g. detailed explanation, stacktraces, related issues, suggestions how to fix, links for us to have context, eg. stackoverflow, etc)

**Comment by H256** [comment]
Trying to get a working docker image from this repo. This might take some time, since I do not have any experience with Docker atm.

**Comment by H256** [comment]
@ethcar I have altered the `docker-entrypoint.sh`, so that it points to the correct paths. Could you `git pull` and give it a try? Maybe it already works this way.

**Comment by ethcar** [comment]
Hi, I myself tried doing this before, it did not work,
After checking your msg I tried again with your repo, unfortunately it did not help, I even updated the package.json for the required dependencies and dev dependencies listed in quasar UI package,

If you want hyou can try running on your server and check, if you would like I can create a test server and give you the credentials if you would like to try.  
Thanks for your help. I will also open issue in gekko repo for server not working, I have tried every method, different files, Only via docker it works, you can give it a try at your end too..

**Comment by ethcar** [comment]
Hi, I have got upto this point, refer to the screenshot: http://prntscr.com/mn73id

UI  opens however screen is disconnected. 

http://134.209.4.166:3002

**Comment by H256** [comment]
@ethcar Getting closer :) Looks like the socket cannot establish a connection. If you edit the UiConfig.js in web/vue/statics and change the ui ip adress to your server ip, it might work. 

I can create an example as soon as I have access to a computer again ;)

**Comment by ethcar** [comment]
Hey, Thanks for your time.
It is working with custom UI. 
I have been trying to figure it out from past few days, I should have contacted you earlier.

**Comment by ethcar** [comment]
Hi, 
I have been running the UI, and it works fine, however I just noticed that for any strategy I use History size is always 1 minute, I setup multiple watchers and and ran the papertrader.

Is it okay, or just a viual glich because I always fill the parameter to use history size of 1 or 2 hours atleast.
I noticed it today. To test this I am will run a new instance of old UI with same parameters and check.

If there is bug, and if you get time to fix it, let me know.

Thank you for your time and help
Screenshot:
http://prntscr.com/mnsjpp
